.env.dist

```diff
@@ -31,4 +31,12 @@ BRUINCAST_API=
 VIDEORES_FTP_HOST=
 VIDEORES_FTP_USERNAME=
 VIDEORES_FTP_PWD=
-VIDEORES_FILEPATH=
\ No newline at end of file
+VIDEORES_FILEPATH=
+# Tab constants
+TAB_BRUINCAST=0
+TAB_VIDEO_RESERVES=1
+TAB_DIGITAL_AUDIO_RESERVES=2
+TAB_MEDIA_GALLERY=3
+TAB_ADMIN_PANEL=4
+TAB_ADMIN_PANEL_SETTINGS=0
+TAB_ADMIN_PANEL_LISTINGS=1
```

Comment:
Is there a reason the constants are .env? I thought we already had them in constants.js